mrmustard/lab_dev/states/base.py

```diff
@@ -829,6 +829,21 @@ def __getitem__(self, modes: Union[int, Sequence[int]]) -> State:
             representation, wires, self.name
         )  # pylint: disable=protected-access
 
+    @classmethod
+    def random(cls, modes, max_r=1.0):
```

Comment:
Can we make `m` a parameter (that defaults to `len(m)`? That way one can make the DM more/less mixed by tracing a smaller/larger ancilla.

Reply:
Sure thing!

Reply:
Should I do the same with Channels?